compiler/src/main/java/com/squareup/anvil/compiler/AnvilCommandLineProcessor.kt

```diff
@@ -80,3 +80,18 @@ class AnvilCommandLineProcessor : CommandLineProcessor {
     }
   }
 }
+
+class CommandLineOptions private constructor(
```

Comment:
What do you think about moving this to its own file? Alternatively, if we bundle it in the file with another class, it feels like bundling it with `CompilerConfiguration` might be better if we're enforcing that it can only be instantiated from that object.

Reply:
That's fair.